apps/ejabberd/src/node_push.erl

```diff
@@ -207,20 +208,16 @@ is_allowed_to_publish(PublishModel, Affiliation) ->
 parse_form(undefined) ->
     #{};
 parse_form(Form) ->
-    IsForm = ?NS_XDATA == exml_query:attr(Form, <<"xmlns">>),
-    IsSubmit = <<"submit">> == exml_query:attr(Form, <<"type">>),
```

Comment:
Hi @astro,

I understand this `type` check was an issue for you. Could you give us more details? What implementation do you have in mind?